README.md

```diff
@@ -17,55 +17,114 @@ For further details about the method, please refer to the [PDF documentation](ht
+We don't ship binaries at the moment, so to use TDMS, it has to be compiled.
+It needs to be built against [FFTW](https://www.fftw.org/) and [MATLAB](https://www.mathworks.com/products/matlab.html), which must be downloaded and installed first.
 
 <details>
-<summary>Mac-specific instructions</summary>
+<summary><img src="https://github.com/EgoistDeveloper/operating-system-logos/blob/master/src/24x24/UBT.png"/> Linux prerequisite setup</summary>
 
-To compile TDMS on a Mac, you will need an x86 compiler with libraries for OpenMP.
-You can install these using [Homebrew](https://brew.sh) with the command:
+Assuming you don't already have them, you'll need a C++ compiler, CMake, OpenMP and FFTW.
 
+For Debian-based distributions this should be as simple as
 ```{sh}
-$ brew install llvm
+$ sudo apt install git gcc cmake libfftw3-dev libgomp1
 ```
 
-After installing with Homebrew, you may need to set the following CMake arguments:
+</details>
+
+<details>
+<summary><img src="https://github.com/EgoistDeveloper/operating-system-logos/blob/master/src/24x24/MAC.png"/> MacOS prerequisite setup</summary>
+
+On MacOS you will need an x86 compiler with libraries for OpenMP.
+You'll need to download the latest [Xcode tools](https://apps.apple.com/app/xcode).
+
+Everything else can be installed using [Homebrew](https://brew.sh):
 
 ```{sh}
--DCMAKE_CXX_COMPILER=/Users/username/.local/homebrew/opt/llvm/bin/clang++
--DOMP_ROOT=/Users/username/.local/homebrew/opt/llvm/
--DCXX_ROOT=/Users/username/.local/homebrew/opt/llvm
--DHDF5_ROOT=/Users/username/.local/homebrew/opt/hdf5
+$ brew install cmake fftw llvm
 ```
 
-On an ARM Mac, you will need to install the x86 version of Homebrew.
-To do so, use the following commands:
+On an ARM Mac, you will need to install the x86 version of Homebrew:
 
 ```{sh}
 $ arch -x86_64 zsh
 $ arch -x86_64 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
-$ arch -x86_64 /usr/local/bin/brew install llvm hdf5
+$ arch -x86_64 /usr/local/bin/brew install cmake fftw llvm
+```
+
+</details>
+
+<details>
+<summary><img src="https://github.com/EgoistDeveloper/operating-system-logos/blob/master/src/24x24/WIN.png"/> Windows prerequisite setup</summary>
+
+TDMS was developed on, and has been extensively tested on linux.
+Support for Windows is quite new and experimental (please [report](https://github.com/UCL/TDMS/issues/new/choose) any issues you encounter!).
+
+It might be more straightforward to use the [Windows subsystem for Linux (WSL2)](https://learn.microsoft.com/en-gb/windows/wsl/install), or set up an linux virtual machine.
+
+However, TDMS _can_ be compiled natively on Windows.
+This has been tested Windows 10 with PowerShell.
+
+Assuming you don't already have them, you'll need to download and install:
+
+* [MATLAB](https://www.mathworks.com/products/matlab.html),
+* [Visual Studio](https://visualstudio.microsoft.com/vs/community/) and be sure to select the C++ kit,
```

Comment:
```suggestion
* [Visual Studio](https://visualstudio.microsoft.com/vs/community/) and **be sure to select the C++ kit**,
```

I feel like this should carry this emphasis since if you're like me (or the students that attend Carpentries workshops), you see a list of prerequisites and you don't look past the link you're given to the installer. Bold text helps draw the eye to "there's more to this step than you think".